### DISCUSSION TITLE
I made a simple graph of average regional variation per individual for Wikipedia from your data

### DISCUSSION THREAD

**Discussion by clawsoon** [tskit-dev/tskit#1793] [Show and tell]
This is a wee graph I made from all your data for a couple of Wikipedia articles on human genetic variation:

https://en.wikipedia.org/wiki/Human_genetic_variation#/media/File:Human_genetic_variant_counts_by_region.svg

**Comment by petrelharp** [comment]
Very nice - you should probably say what dataset was used in the legend?

**Reply by clawsoon** [reply]
Good point.  Would it make more sense to link to the paper on Biorxiv or to the dataset on Zenodo?

**Reply by petrelharp** [reply]
Probably zenodo, if that's got a concise description of where the data were compiled from?

**Reply by clawsoon** [reply]
I ended up using the paper link, since Wikipedia has a citation template for Biorxiv and I was too lazy to figure out how to link to Zenodo properly, lol.
